examples/io_perf_test.cpp

```diff
@@ -0,0 +1,176 @@
+#include <iostream>
```

Comment:
It seems like "io_perf_test.cpp" is maybe more of a test than an example? There is the beginning of some C++ tests in the "tests" directory.

Reply:
Well, except that it requires a path to an .otio file, and then profiles the time to read it in and out with and without downgrading. So it's a bit between example, profile harness, and test. The upgrade/downgrade example program is a more straightforward example, I agree. But I've been using it as a mix of correctness and performance test with big production files.

Reply:
Hmm, still sounds like a test to me but fair enough. :) Probably a good idea to add the SPDX headers to this and the other example.

Reply:
Good note!  My general policy is to try and capture these kinds of interstitial programs and put them somewhere in the repo, especially since (As you had previously noted) we don't have a ton of C++ example code.